### ISSUE TITLE
Require Qt6.6+

### ISSUE THREAD

**Issue by edreamleo** [leo-editor/leo-editor#2882]
See PR #2883. Closed  via #3828.

Supporting only Qt6 allows better mypy annotations.

**Comment by tbpassin** [comment]
As I said in my post on this on Google Groups, we should go very slowly with this, otherwise our Linux users are likely to have real problems.

**Comment by edreamleo** [comment]
@tbpassin Thanks for your comments.  

[This page](https://www.pythonguis.com/installation/install-pyqt6-linux/) says that this works: `pip3 install pyqt6`.   Why is this a big deal?

**Comment by SegundoBob** [comment]
Edward,

I am now using Ubuntu 20.04 and Python 3.8.10, PyQt version 5.12.8.

I'm considering moving to Ubuntu 22.04 and PyQt6, but I know this won't be easy.   In fact, so far as I can tell, both moves would be premature at this time.  For example, I tried to pip install PyQt6 and the install failed (nothing was installed):
```

2022-09-19 14:14:50 /pri/git/sqlite_enter
$ pip3 install --user PyQt6
Collecting PyQt6
  Downloading PyQt6-6.3.1-cp37-abi3-manylinux1_x86_64.whl (7.8 MB)
     |████████████████████████████████| 7.8 MB 4.8 MB/s 
Collecting PyQt6-sip<14,>=13.4
  Downloading PyQt6_sip-13.4.0-cp38-cp38-manylinux1_x86_64.whl (310 kB)
     |████████████████████████████████| 310 kB 21.3 MB/s 
ERROR: Could not find a version that satisfies the requirement PyQt6-Qt6>=6.3.0 (from PyQt6) (from versions: none)
ERROR: No matching distribution found for PyQt6-Qt6>=6.3.0 (from PyQt6)
```
Respecfully,
SegundoBob

**Comment by edreamleo** [comment]
@SegundoBob Thanks for your comments.

I too have been unable to upgrade to Ubuntu 22.04.

I'm convinced that Leo can't require Qt6 for now. Maybe next year. Maybe not.

Happily, PR #2882 may uncover dubious code. I suspect Leo 6.7.1 will benefit, despite supporting both Qt5 and Qt6.

**Comment by edreamleo** [comment]
Closed via PR #3828.
